bittensor/utils/networking.py

```diff
@@ -82,8 +81,8 @@ def ip__str__(ip_type:int, ip_str:str, port:int):
 class ExternalIPNotFound(Exception):
     """ Raised if we cannot attain your external ip from CURL/URLLIB/IPIFY/AWS """
 
-def get_external_ip() -> str:
-    r""" Checks CURL/URLLIB/IPIFY/AWS for your external ip.
+def get_external_ip(timeout:int = 12) -> str:
```

Comment:
```suggestion
def get_external_ip(timeout:int = bittensor.__blocktime__) -> str:
```
Use blocktime here too?